cartography/intel/github/util.py

```diff
@@ -70,6 +70,14 @@ def call_github_api(query: str, variables: str, token: str, api_url: str) -> Dic
         # Add context and re-raise for callers to handle
         logger.warning("GitHub: requests.get('%s') timed out.", api_url)
         raise
+
+    if response.status_code == 502:
+        logger.error(
```

Comment:
```suggestion
        logger.warning(
```
Github API errors are logged as warning above/below this code.

Reply:
+1. error-level logs in cartography are used to indicate fatals.